src/XMLimport.cpp

```diff
@@ -993,8 +993,8 @@ void XMLimport::readHostPackage(Host* pHost)
             } else if (name() == "mLightWhite") {
                 pHost->mLightWhite.setNamedColor(readElementText());
             } else if (name() == "mDisplayFont") {
-                pHost->mDisplayFont.fromString(readElementText());
-                pHost->mDisplayFont.setFixedPitch(true);
+                pHost->setDisplayFont(readElementText());
```

Comment:
What happens if the profile had (for some reason) a 0 width font set already?

Reply:
it would have crashed before it could have been saved? :boom: :man_shrugging:

Reply:
Yeah. But I'll fix it anyway to reset to default in case someone hand edited the save file.

Reply:
Fixed.